source/src/world.h

```diff
@@ -76,7 +76,7 @@ struct mapstats
             if(!e.attr2) e.attr2 = 255; /* needed for MAPVERSION<=2 */ \
             if(e.attr1>32) e.attr1 = 32; /* 12_03 and below */ \
         } \
-        if(headr.version<6  && strncmp(headr.head,"CUBE",4)==0)  /* only render lights, pl starts and map models on old maps // <6 was <MAPVERSION but we're now at #7 (flowtron) */ \
+        if(strncmp(headr.head,"CUBE",4)==0)  /* only render lights, pl starts and map models on old maps // <6 was <MAPVERSION but we're now at #7 (flowtron) */ \
```

Comment:
If pCube maps have a version number of 6, wouldn't it make sense to change `headr.version<6` to `headr.version<=6`?